src/components/Api.js

```diff
@@ -0,0 +1,179 @@
+export class Api {
+  constructor(config) {
+    this.url = config.baseUrl;
```

Comment:
Needs correction: Variables, which are used only inside the class should be private (add a prefix `_`)

Reply:
I updated all the URLs by adding the prefix.